GameOfLife/Calculations/CellGenerationIterator.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GameOfLife
+{
+    public class CellGenerationIterator
+    {
+       private int iterationCounter = 0;
```

Comment:
Alignment for this line, comment for method, remove unnecessary lines.

Reply:
Removed empty lines. Add summary for method. Fixed alignment.